### PR TITLE
Searching algorithm markdown added

### LINKED ISSUE #1: Searching Algorithms
### Subtopic:

- Linear Search
- Binary Search
- Ternary Search

#### Checks:

- [ ] explanation
- [ ] implementation
- [ ] references
- [ ] extra problems for practice

### PR DESCRIPTION
**Pull request by prerna2712**
# Description

Searching Algorithms

Fixes #1 

## Type of change

Please delete options that are not relevant.

- [x] Added Algorithm
- [x] Folder Structure

# Added to SUMMARY.md (delete section if not needed)?

Have you added the page to SUMMARY.md if this is a new page and not a fix?

- [x] Yes

### CODE REVIEW

File: src/Searching_Algos/Ternary_search/Explanation.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+> \/ &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\/&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\/&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\/
+> <p>1 &nbsp;&nbsp; | &nbsp;&nbsp; 2 &nbsp;&nbsp; | &nbsp;&nbsp; 3 &nbsp;&nbsp; | &nbsp;&nbsp; 4 &nbsp;&nbsp; | &nbsp;&nbsp; 5 &nbsp;&nbsp; | &nbsp;&nbsp; 6 &nbsp;&nbsp; | &nbsp;&nbsp; 7 &nbsp;&nbsp; | &nbsp;&nbsp; 8 &nbsp;&nbsp; | &nbsp;&nbsp; 9 &nbsp;&nbsp; | &nbsp;&nbsp; 10</p>
```
Comment (raydwaipayan) [inline src/Searching_Algos/Ternary_search/Explanation.md:25]: Is this part okay? Have you checked it?

File: src/Arrays/problems.md [hunk 2]
```diff
@@ -1,11 +0,0 @@
-# Recommended LeetCode questions
-- Two Sum
-- Best Time to Buy and Sell Stock
-- Contains Duplicate
-- Product of Array Except Self
-- Maximum Subarray
-- Maximum Product Subarray
```
Comment (raydwaipayan) [inline src/Arrays/problems.md:7]: Was this file created by you or was it there in the repo?

**Review by raydwaipayan — CHANGES_REQUESTED** [review]

File: src/Arrays/problems.md (hunk 2 above)
Comment (prerna2712) [reply src/Arrays/problems.md:7]: This was already there, I guess

File: src/Searching_Algos/Ternary_search/Explanation.md (hunk 1 above)
Comment (prerna2712) [reply src/Searching_Algos/Ternary_search/Explanation.md:25]: Yeah it was working fine

File: src/Arrays/problems.md (hunk 2 above)
Comment (raydwaipayan) [reply src/Arrays/problems.md:7]: > This was already there, I guess

Maybe add it back then.

**Comment by raydwaipayan** [comment]
CI Build error:
0.01s$ mdbook build
2020-11-22 19:46:15 [WARN] (mdbook::book::summary): Expected a start of a link, actually got Some(Text(Borrowed("Searching Algorithms")))
2020-11-22 19:46:15 [ERROR] (mdbook::utils): Error: Summary parsing failed
2020-11-22 19:46:15 [ERROR] (mdbook::utils): 	Caused By: There was an error parsing the numbered chapters
2020-11-22 19:46:15 [ERROR] (mdbook::utils): 	Caused By: There was an error parsing the numbered chapters
2020-11-22 19:46:15 [ERROR] (mdbook::utils): 	Caused By: failed to parse SUMMARY.md line 7, column 3: The link items for nested chapters must only contain a hyperlink

Please resolve

**Review by raydwaipayan — APPROVED** [review]
Thanks Prerna

**Comment by arnabsen1729** [comment]
@allcontributors add @prerna2712  for contributing to the documentation

**Pull request merged**